opba-protocols/xs2a-protocol-tests/xs2a-bdd-tests-common/src/main/java/de/adorsys/opba/protocol/xs2a/tests/e2e/stages/CommonGivenStages.java

```diff
@@ -56,11 +54,10 @@ public SELF rest_assured_points_to_opba_server(String opbaServerUri) {
     }
 
     public SELF user_registered_in_opba_with_credentials(String username, String password) {
-        RestAssured
+        withSignatureHeaders(RestAssured
```

Comment:
This function is called within the user session (from UI), not Fintech - it is protected only by users' cookie and not by request signing - no signature headers here

Reply:
Removed signature header adding for non-fintech requests